### PR TITLE
Update index.page.tsx

### LINKED ISSUE #660: [📝 Docs]: Improve wording of the new Welcome page for the release 3 of the website
### Is your feature request related to a problem? Please describe

We have created a new welcome page for our docs but we need to add all the texts with something inspiring.

You can see that page here: https://web-release-3.website-2v2.pages.dev/overview/welcome

Source: https://github.com/json-schema-org/website/tree/web-release-3/pages/overview/welcome

Branch: https://github.com/json-schema-org/website/tree/web-release-3

### Describe the solution you'd like

Texts that inspires.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by kwennB**
What kind of change does this PR introduce?
Enhancement to documentation

Issue Number:
Closes #660

Summary
Removed the placeholders texts for the newly created welcome page in the release 3.
[json-schema-org#660](https://github.com/json-schema-org/website/issues/660)

Does this PR introduce a breaking change?
No

### CODE REVIEW

**Review by DhairyaMajmudar — COMMENTED** [review]
Hello, @kwennB can you pls. once have a look at that failing lint check.

Thank You : )

**Review by DhairyaMajmudar — APPROVED** [review]

**Review by benjagm — CHANGES_REQUESTED** [review]
Hi @kwennB . Nice job!

As discussed please adjust the tone to better fit our personas.

File: pages/overview/welcome/index.page.tsx [hunk 1]
```diff
@@ -66,9 +66,11 @@ export default function Welcome() {
...
       <Headline1>{newTitle}</Headline1>
       <p>
-        Veniam ea fugiat exercitation laboris non est nulla id pariatur ex. Qui
-        occaecat fugiat sunt exercitation adipisicing culpa reprehenderit
-        consectetur amet in. Qui fugiat amet do eu.
+        JSON Schema is a powerful tool for validating the structure of JSON
+        data. It effectively helps you to annotate and validate the structure,
+        constraints, and data types of your JSON documents. Our goal is to
+        provide a standardized means for you to define the expectations of your
+        JSON data.
```
Comment (benjagm) [inline pages/overview/welcome/index.page.tsx:73]:
```suggestion
        Discover everything you can do with JSON Schema. This section presents
        the most common use cases for JSON Schema, but the possibilities are
        endless.
```

File: pages/overview/welcome/index.page.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function Welcome() {
     {
       title: 'Other Resources',
       summary:
-        'Deserunt et fugiat do adipisicing enim in nostrud Lorem anim ut amet enim.',
+        'Explore our different resources, books, articles, courses, videos, podcasts, and papers, we welcome you to enjoy them.',
```
Comment (benjagm) [inline pages/overview/welcome/index.page.tsx:54]:
```suggestion
        'Deserunt et fugiat do adipisicing enim in nostrud Lorem anim ut amet enim.',
```

Please don't modify the resources description. That section is going to be removed from release 3.

**Review by benjagm — CHANGES_REQUESTED** [review]
I just left small suggestions before getting this merged.

**Review by benjagm — APPROVED** [review]
Nice job Blessing!

**Pull request merged**